### ISSUE TITLE
Email from feedrabbit.com marked as non-article

### ISSUE THREAD

**Issue by edshamis** [omnivore-app/omnivore#1917]
and bounced to my Gmail
![Screenshot_2023-03-15-07-06-21-084_com.android.chrome.jpg](https://user-images.githubusercontent.com/47914518/225212864-797a2546-8b6b-4e2b-948b-2e52ea928191.jpg)

**Comment by edshamis** [comment]
@jacksonh I keep getting emails from Omnivore every few hours for the same newsletter marked as non-article
![Screenshot_2023-03-16-22-29-28-267_com.google.android.gm.jpg](https://user-images.githubusercontent.com/47914518/225745608-f90f2f4f-8dd8-436f-b35f-9c75dfc60bc5.jpg)

**Comment by jacksonh** [comment]
This is quite strange like it's either being retried on our side or theirs. Looking into the logs now.

**Comment by jacksonh** [comment]
@edshamis it seems we are receiving an email from them every hour, we don't detect it as an article, and then forward it to you.
<img width="324" alt="Screen Shot 2023-03-17 at 07 47 54" src="https://user-images.githubusercontent.com/75189/225777008-0f88d3d9-9c29-4f28-9029-311cabf4db96.png">

Should these emails be marked as articles?

**Comment by edshamis** [comment]
@jacksonh yes and only once I suppose. Btw I unsubscribed from rss on their side but these emails are sill coming...

**Comment by jacksonh** [comment]
@edshamis ok, let me check and see if they are still sending them.

**Comment by jacksonh** [comment]
@edshamis it looks like they should have stopped about two hours ago based on our logs. 

I don't think we can handle what you were trying to do here, for each email we receive we either add it to the library or forward it, we can't quite compress them together with previous entries.
